test/charms/test_main/mod/operator/ops

```diff
@@ -0,0 +1 @@
+../../../../../ops
```

Comment:
I'm happy to see the rename lib/charm.py to src/charm.py, but I'm not sure that it is worth introducing mod just to have a symlink pointing to a symlink.

Reply:
Created a new PR as I forgot to create a branch for this one https://github.com/canonical/operator/pull/112